### PR TITLE
Support for post mortems (aka "Trainer experiences")

### PR DESCRIPTION
**Pull request by erasche**
Chose to name them something friendlier than we use in operations, the only place 'post mortem' remains is in the branch.

Added to topic pages as "For trainers"

grafik

The trainer experience list:

grafik

An example trainer experience:

grafik

### CODE REVIEW

**Comment by erasche** [comment]
@tklingstrom how do you feel about this as well? The sort of thing you imagined?

**Comment by TKlingstrom** [comment]
Yes, I really like it.

File: _layouts/training_experience.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+</header>
+
+<div class="container main-content">
+    <section class="tutorial">
+        <h1>{{ page.title }}</h1>
```
Comment (bebatut) [inline _layouts/training_experience.html]: Maybe adding the date also?

File: _layouts/training_experience_list.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+</header>
+
+<div class="container main-content">
+    <section class="tutorial">
+        <h1>{{ page.title }}</h1>
```
Comment (bebatut) [inline _layouts/training_experience_list.html]: There was no title displayed for me locally. Maybe we could have a default title there

**Comment by erasche** [comment]
Probably in instructors is a good place. I'll add some documentation for that.

But @shiltemann has some really good points that we should discuss (maybe in person). The main one is: many people give many trainings and it would be hard to ask them to split it into 6 different documents (for each training), and it leaves them with no place for "overall stuff" unless they're integrating that into the "training todo list".

(Any offline discussion I will summarize discussion here for future reference.)

**Comment by erasche** [comment]
This really feels sufficiently implemented by just reporting in the workshop  feedback issue #1452.

Closing this, we don't need it more structured or having a higher threshold to contribute feedback.

**Pull request closed without merging**